onnx2kerastl/linear_layers.py

```diff
@@ -31,24 +33,26 @@ def convert_gemm(node, params, layers, lambda_func, node_name, keras_name):
     if 'transB' in params and params['transB'] == 1:
         logger.debug('Transposing W matrix.')
         keras_weights[0] = keras_weights[0].transpose()
-
     # Estimate input/output neurons
-    input_channels, output_channels = keras_weights[0].shape
+    input_channels, output_channels = keras_weights[0].shape[-2:]
     logger.debug('Input units %s, output units %s.', input_channels, output_channels)
+    if len(keras_weights[0].shape) > 3: #N-dim tensor multipication Dense doesn't work
+        assert len(node.input) == 2
+        layers[node_name] = tf.matmul(layers[node.input[0]], layers[node.input[1]], name=keras_name)
```

Comment:
As far as i know Dense layer can perform N-dim multipications

Reply:
Things is we multiple Nd matrices here (no weights) - will change the condition from 3 to 2